README.md

```diff
@@ -2,11 +2,25 @@
 
 This project was generated using [Nx](https://nx.dev).
 
-## Build
+### [pdf-renderer](https://github.com/leancodepl/pdf-generator/tree/main/packages/pdf-renderer)
```

Comment:
:thinking: wondering if the url here shouldn't be relative. So if for example you're looking at readme on one branch you don't get redirected always to main.